synse/http.go

```diff
@@ -176,3 +375,41 @@ func check(err error, errResp *Error) error {
 
 	return nil
 }
+
+// makeURI joins the given components into a string, delimited with '/' which
+// can then be used as the URI for API requests.
+func makeURI(components ...string) string {
+	return strings.Join(components, "/")
+}
```

Comment:
This looks good for now, but in the future, we may want to look into leveraging something like: https://golang.org/pkg/net/url/

In particular, I think that would be useful when looking at adding https support (so, further down the line). I haven't looked at the package extensively, but it may have some of these url building utils in it that we could leverage, perhaps.